diff -up expect-5.44.1.15/expect.man.orig expect-5.44.1.15/expect.man
--- expect-5.44.1.15/expect.man.orig	2012-01-26 11:55:05.491604479 +0100
+++ expect-5.44.1.15/expect.man	2012-01-26 11:55:51.681711911 +0100
@@ -173,7 +173,8 @@ way, use the
 .B \-b
 flag.
 (When using Expectk, this option is specified as
-.BR \-buffer .)  Note that stdio-buffering may still take place however this shouldn't cause problems when reading from a fifo or stdin.
+.BR \-buffer .)
+Note that stdio-buffering may still take place however this shouldn't cause problems when reading from a fifo or stdin.
 .PP
 If the string "\-" is supplied as a filename, standard input is read instead.
 (Use "./\-" to read from a file actually named "\-".)
