diff -up expect-5.44.1.15/example/passmass.orig expect-5.44.1.15/example/passmass
--- expect-5.44.1.15/example/passmass.orig	2012-01-26 11:58:01.546825504 +0100
+++ expect-5.44.1.15/example/passmass	2012-01-26 11:58:29.604498207 +0100
@@ -167,7 +167,7 @@ for {set i 0} {$i<$argc} {incr i} {
     }
 
     if ($su) {
-	send "su -\r"
+	send "/bin/su -\r"
 	expect -nocase "password:"
 	send "$password(old)\r"
 	expect "# "
